build.gradle

```diff
@@ -13,12 +13,18 @@ subprojects {
 
     repositories {
         mavenCentral()
+        maven { url 'https://jitpack.io' }
     }
 
+    apply plugin: "java"
     apply plugin: "checkstyle"
     apply plugin: "pmd"
     apply plugin: "com.github.spotbugs"
     apply plugin: "codenarc"
+    
+    dependencies {
+        implementation 'com.github.papaya-agh:papaya-interface:IO2019TEAM-68-SNAPSHOT'
```

Comment:
Pewna jesteś co do tego? Raczej tylko `papaya-web` powinien zależeć od API.

Reply:
Absolutnie nie jestem pewna

Mogę zostawić te dwa enumy bez żadnego mapowania między nimi na razie, ktoś to potem ogarnie w `papaya-web`

Reply:
właśnie na ten temat toczy się dyskusja xd